src/Forces.jl

```diff
@@ -0,0 +1,66 @@
+using OrderedCollections
+"""
+A simple struct to contain a vector of force vectors, and utilities to print them in a nice format.
+"""
+struct Forces{T <: Union{Nothing, AbstractVector}}
+    # energies["TermName"]
+    # parametrization on T acts as a nice check that all terms return correct type
+    forces::OrderedDict{String, T}
```

Comment:
OrderedDict has been a bad choice for the energies already. Please don't use it here. Just make `Forces` behave like an `AbstractDict` and make this a simple list (One list of string and one of T).